src/frontend/components/header/header.html

```diff
@@ -51,7 +51,7 @@ <h4 class="regular flex flex-center primary-color m0">
   </header>
 
   <form>
-    <ul class="setting-dropdown list-reset border border-dark rounded bg-silver absolute right-0" *ngIf="settingOpened">
+    <ul class="setting-dropdown list-reset border border-dark rounded bg-silver absolute right-0" [ngClass]="{'setting-dropdown--dark':theme==='dark'}" *ngIf="settingOpened">
```

Comment:
There are many ways of add/removing a css class in ng2, I like this one the best. With that said, I am open to other ways of achieving the same result.